.github/workflows/test-builds-linux.yml

```diff
@@ -0,0 +1,102 @@
+name: Linux Test Builds
```

Comment:
I would shorten this to maybe just "CI", because the individual job names in the file will be appended.